org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/ASTConverter.java

```diff
@@ -3630,15 +3630,35 @@ public TypeParameter convert(org.eclipse.jdt.internal.compiler.ast.TypeParameter
 		return typeParameter2;
 	}
 
+	@SuppressWarnings("cast")
```

Comment:
what problem is being suppressed here?

Reply:
Done. I have removed the SuppressWarnings